js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -919,6 +921,13 @@ export const AuctionCard = ({
           to redeem their bids for them right now.
         </h3>
       </MetaplexModal>
+      <CongratulationsModal
+        isModalVisible={isOpenCongratulations === 'purchase'}
+        onClose={() => setIsOpenCongratulations('')}
+        onClickOk={() => window.location.reload()}
```

Comment:
Can we avoid Reload like this?

Reply:
I noticed that such a reload is used in the project, not sure that we have else way. But I can remove this so user reload page manually.

Reply:
Okay let it be